include/SupportInfillPart.h

```diff
@@ -33,7 +33,7 @@ class SupportInfillPart
                                                                             //   for infill_areas[x][n], x means the density level and n means the thickness
     std::vector<VariableWidthLines> wall_toolpaths; //!< Any walls go here, not in the areas, where they could be combined vertically (don't combine walls). Binned by inset_idx.
 
-    coord_t custom_line_distance;
+    coord_t custom_line_distance;//!< The distance between support infill lines. 0 means use the default line distance instead.
```

Comment:
there should be a space between a statement and a comment
```suggestion
    coord_t custom_line_distance; //!< The distance between support infill lines. 0 means use the default line distance instead.
```